cime/src/drivers/moab/cime_config/config_compsets.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0"?>
+
+<compsets version="2.0">
+
+  <help>
+    =========================================
+    compset naming convention
```

Comment:
I assume most if not all of this XML was copied from mct driver. I'm worried about things falling out of sync when mct changes. I would prefer symlinks for files that are exactly the same.

We should maybe think about providing a system for overriding individual values from mct so we don't need to copy the whole file.

Reply:
I'm not sure how many of these files will still be around when we're done. I'll switch to symlinks and we'll remove or replace them as needed.

Reply:
Yes we're not touching the XML system so should just be able to copy any further changes.  Or use symlinks.  Or maybe this info should be in its own directory.  We can work it out later.